docs/fides/docs/tutorial/add.md

```diff
@@ -0,0 +1,76 @@
+# Add Fidesctl to the App
+
+In this step you will incorporate the [Fidesctl](https://github.com/ethyca/fides) open-source developer tools, which will enable you to declare your `system`, `dataset`s, and `policy`s as manifest YAML files later in this tutorial.
+
+## Add the `fidesctl` Dependency
+
+Open the `requirements.txt` file and add the `fidesctl` dependency by including the following line:
+
+```
+fidesctl>=1.0.0
+```
+
+## Configure Fidesctl
+
+Create a `fidesctl.toml` file at the root level of the repository. It should contain the following configuration:
+
+```toml
+[cli]
+server_url = "http://localhost:8080"
+
+[api]
+database_url = "postgresql://postgres:postgres@localhost:5432/fidesctl"
+```
+
+- The `[cli]`-scoped `server_url` option specifies the address that the Fidesctl CLI will use when connecting to the Fidesctl server.
+- The `[api]`-scoped `database_url` option specifies the connection string that the Fidesctl API will use to connect to the PostgreSQL database created in the previous step.
+
+## Run Fidesctl via Docker
+
+Since the app uses `docker-compose` to orchestrate resources, include `fidesctl` as a service by adding the following configuration after the database service:
```

Comment:
Add another establishing sentence here like:
> Next, we need to get the fidesctl server up and running. Since this app uses `docker-compose` we can just...

Also probably good to link to the deployment guide for more detail: https://ethyca.github.io/fides/deployment/#step-2-setup-the-fidesctl-web-server